packages/client/libclient-py/quickmpc/qmpc_server.py

```diff
@@ -88,6 +90,38 @@ def _argument_check(join_order: Tuple[List, List, List]):
             return False
         return True
 
+    def __retry(self, f: Callable, *request: Any) -> Any:
+        for _ in range(self.retry_num):
+            try:
+                return f(*request)
+            except grpc.RpcError as e:
+                logger.error(f'{e.details()} ({e.code()})')
+
+                # エラーが詳細な情報を持っているか確認
+                status = rpc_status.from_call(e)
+                if status is not None:
+                    for detail in status.details:
+                        if detail.Is(
+                            # type: ignore[attr-defined]
+                            JobErrorInfo.DESCRIPTOR
```

Comment:
`type ignore` must be on the right side of the line.
However, since it is no longer necessary to ignore, please delete it this time.
ref: https://mypy.readthedocs.io/en/stable/error_codes.html#silencing-errors-based-on-error-codes
```suggestion
                            JobErrorInfo.DESCRIPTOR
```

Reply:
[d98e225](https://github.com/acompany-develop/QuickMPC/pull/154/commits/d98e2255c681cbef99dc57e165a84569a8318461)